python/tutorials/06-fused-attention.py

```diff
@@ -469,6 +473,16 @@ def forward(ctx, q, k, v, causal, sm_scale):
                     BLOCK_N = 128
                     num_stages = 3
                     num_warps = 8
+        # Tuning for AMD backend
```

Comment:
I'd put this before L455, and restructure it as:

```python
if is_hip(): ..
elif torch.cuda.get_device_capability()[0] == 9: // h100
else: // other cuda targets
```
to be clear. they are disjoint cases.